### ISSUE TITLE
limit

### ISSUE THREAD

**Issue by Grandiz** [ryanfitz/vogels#187]
Here is my query:

```
var scan = Message
                        .scan()
                        .filterExpression('#sender = :userId OR #recipient = :userId')
                        .expressionAttributeValues({':userId': userId})
                        .expressionAttributeNames({'#sender': 'sender', '#recipient': 'recipient'});

                    if (paginationLimit != 0) {
                        scan.limit(paginationLimit);
                    }

                    if (e.params.querystring.startKey) {
                        scan.startKey(e.params.querystring.startKey);
                    }
                    scan.exec(finish);
```

The result is Count: 2, ScannedCount: 10. But in table i have 3 entity with same userId as i passed. 
Scan below return Count: 3

```
Message
                        .scan()
                        .filterExpression('#sender = :userId OR #recipient = :userId')
                        .expressionAttributeValues({':userId': userId})
                        .expressionAttributeNames({'#sender': 'sender', '#recipient': 'recipient'})
                        .select('COUNT')
                        .exec(finish);
```

Could any one help me?

**Comment by clarkie** [comment]
This question is answered over in dynogels (a fork of vogels): https://github.com/clarkie/dynogels/issues/12

tl;dr - it's expected behaviour.
